index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Team Website</title>
+	<link rel="stylesheet" href="style.css">
+</head>
+<body>
+	<div calss="nav">
+		<nav>
+			<div class="home"><a href="#" class="home-link">Home</a></div>
+			<ul>
+				<li><a href="#">Team</a></li>
+				<li><a href="#">Skills</a></li>
+				<li><a href="#">Contact</a></li>
+			</ul>
+		</nav>
+		
+
+	</div>
+</body>
+</html>
```

Comment:
Add empty line at the end of a file. Always.

This is useful because in node.js we do something called file concatenation, and having no empty line at the end could lead to some bugs that are difficult to track.

Don't worry about understanding the explanation now, I'll explain it better when we get to week 5. For now, just get into the habit of adding new line.